loadbalancer/roundrobin.go

```
@@ -121,7 +127,7 @@ func (rr *RoundRobinLoadBalancer) changeEndpointStatus(connectAddress string, st
 				break
 			}
 		} else {
-			panic("Non-LoadBalancingEndpoint in round robin router endpoint collection")
+			log.Error("Round robin load balancer misconfiguration: non round robin load balancer in round robin pool")
```

Do you want to create and return an error here too instead of just logging it?